function1.js

```diff
@@ -0,0 +1,32 @@
+
+
+function angle (aNum) {
+       if ( aNum === 90) {
+        return "It's a right Angle";
+    } else if (aNum <= 90 && aNum > 0) {
+        return"It's an acute Angle";
+    } else if (aNum == 180) {
+        return"It's a straight Angle";
+    } else if (aNum < 180 && aNum > 90) {
+        return"It's a obtuse Angle";
+    } else if (aNum >180 && aNum < 360) {
+        return"It's a reflex Angle";
+    } else if (aNum == 360) {
+        return "It's a Circle ";
+    } else {
+        return"This is not a valid number";
+    }
+}
```

Comment:
I like the idea for this function! It would be more helpful if it just returned the text "right angle", "acute angle", "straight angle", "reflex angle", "full circle", and "invalid number" or something like that.

That would make it more flexible for the user who is calling the function.